pkg/controllers/dynakube/extension/eec/reconciler.go

```diff
@@ -70,3 +81,57 @@ func (r *reconciler) Reconcile(ctx context.Context) error {
 
 	return r.createOrUpdateStatefulset(ctx)
 }
+
+func (r *reconciler) reconcileTlsSecret(ctx context.Context) error {
+	query := k8ssecret.Query(r.client, r.client, log)
+
+	secret, err := query.Get(ctx, types.NamespacedName{
+		Name:      r.getSelfSignedTlsSecretName(),
+		Namespace: r.dk.Namespace,
+	})
+	if err != nil && !k8serrors.IsNotFound(err) {
+		return err
+	}
+
+	if r.dk.Spec.Templates.ExtensionExecutionController.TlsRefName != "" {
+		if secret != nil {
+			return query.Delete(ctx, secret)
+		}
+	} else {
+		if k8serrors.IsNotFound(err) {
+			// TODO: domain/altNames/ip ?
+			cert, err := certificates.New("", []string{}, "")
```

Comment:
is it something we need to clarify?

cc @StefanHauth

Reply:
More info in this regard here
https://bitbucket.lab.dynatrace.org/users/stefan.hauth/repos/extensions-demo/browse/preview/cert-request.txt
I will try using the same alt names

Reply:
[FIxed](https://github.com/Dynatrace/dynatrace-operator/pull/3778/commits/8f912b88edf5f695f95af496983cced770f5ef14)

Reply:
commit not found https://github.com/Dynatrace/dynatrace-operator/pull/3778/commits/8f912b88edf5f695f95af496983cced770f5ef14

Reply:
Links broke because of force pushes in between, but the fix is there